app/assets/javascripts/folio/cloudflare_turnstile.js

```diff
@@ -0,0 +1,25 @@
+// Global Cloudflare Turnstile
+
+const renderTurnstile = () => {
+  if (!window.turnstileSiteKey || typeof turnstile === 'undefined') return
+
+  if (document.querySelector(".cf-turnstile")) {
+    turnstile.render('.cf-turnstile', {
+      sitekey: window.turnstileSiteKey,
+      appearance: 'interaction-only'
+    })
+  }
+}
+
+const removeTurnstile = () => {
+  if (typeof turnstile === 'undefined') return
+
+  turnstile.remove()
+}
+
+window.onloadTurnstileCallback = () => {
+  renderTurnstile()
+
+  document.addEventListener("turbolinks:load", renderTurnstile)
+  document.addEventListener("turbolinks:before-render", removeTurnstile)
+}
```

Comment:
Nelibi se mi, ze je to rozstrakane na spouste mist. Udelejme prosim `Folio::Security::TurnstileComponent`, kde se tahle logika seskupi. V `render?` bude mit tu `Folio::Security.captcha_provider == :turnstile`, javascript bude pres stimulus. Kdyztak se porad s @davidduben nebo @VladaTrefil